FormalConjectures/ForMathlib/AlgebraicGeometry/ProjectiveSpace.lean

```diff
@@ -0,0 +1,39 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+import Mathlib.AlgebraicGeometry.Limits
+import Mathlib.AlgebraicGeometry.ProjectiveSpectrum.Scheme
+import Mathlib.RingTheory.MvPolynomial.Homogeneous
+
+universe u v
+
+
+open CategoryTheory Limits MvPolynomial AlgebraicGeometry
+
+variable (n : Type v) (S : Scheme.{max u v})
+
+local notation3 "ℤ[" n "]" => homogeneousSubmodule n ℤ
+local notation3 "ℤ[" n "].{" u "," v "}" => homogeneousSubmodule n (ULift.{max u v} ℤ)
```

Comment:
How come you are using notation3 here? Does notation not work?

Reply:
It is needed for the second one (changed the first to `notation`)